### ISSUE TITLE
Unclear what the action should be on "Who can see this" post

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform-pattern-library#16]
We now have a 'change this' button [in the post summary](https://www.dropbox.com/s/ld8ovdx4w43a8pu/Screenshot%202015-11-12%2012.55.22.png?dl=0) and [on post detail](https://www.dropbox.com/s/kfizcquzeabunrl/Screenshot%202015-11-12%2012.55.49.png?dl=0)

Its not clear what the action should be on these? Do we direct to edit post? show a modal? a dropdown?

**Comment by rjmackay** [comment]
cc @willdoran

**Comment by brandonrosage** [comment]
Great catch. I've revised these patterns to include all the necessary markup to complete the "Change," if the user has permission to do so.

Some notes on implementation:

1. I've created a variation to the "custom-select" pattern that, when you add the class name "cta," it makes the value look more like content and the action button more discreet. But it still behaves just like a select control. It's in play on the post summary patterns in the [Home List View](http://github.ushahidi.org/platform-pattern-library/assets/html/5_layouts/home-list.html).

2. The pattern for displaying a post's visibility is different depending on whether the user has permission to change that data. These variations are documented in the HTML for each layout that uses them. ([Post Detail](https://github.com/ushahidi/platform-pattern-library/blob/gh-pages/pattern-library/5_layouts/post-detail.html#L25-L38), [Home List View](https://github.com/ushahidi/platform-pattern-library/blob/gh-pages/pattern-library/partials/_post-listing-with-image.html#L38-L48))

3. I've also added [a custom JavaScript file](https://github.com/ushahidi/platform-pattern-library/blob/gh-pages/assets/js/custom/custom-select.js) that looks at any instance of `.custom-select` and, if any of its `<option>` elements include the `data-icon` attribute, it uses it to keep the select control updated with the appropriate icon for the selected option.
